src/test/java/io/confluent/examples/streams/WordCountLambdaExampleTest.java

```diff
@@ -0,0 +1,137 @@
+/*
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.confluent.examples.streams;
+
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.*;
+import org.apache.kafka.streams.*;
+import org.apache.kafka.streams.test.ConsumerRecordFactory;
+import org.apache.kafka.streams.test.OutputVerifier;
+import org.junit.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Stream processing unit test of {@link WordCountLambdaExample}, using TopologyTestDriver.
+ *
+ * @author Jukka Karvanen / jukinimi.com
+ *
+ * See {@link WordCountLambdaExample} for further documentation.
+ *
+ * See {@link WordCountLambdaIntegrationTest} for the End-to-end integration test using an embedded Kafkacluster.
+ *
+ * Note: This example uses lambda expressions and thus works with Java 8+ only.
+ */
+public class WordCountLambdaExampleTest {
+
+  private TopologyTestDriver testDriver;
+
+  private StringDeserializer stringDeserializer = new StringDeserializer();
+  private LongDeserializer longDeserializer = new LongDeserializer();
+  private ConsumerRecordFactory<String, String> recordFactory = new ConsumerRecordFactory<>(new StringSerializer(), new StringSerializer());
+
+  @Before
+  public void setup() throws IllegalAccessException, ClassNotFoundException, InstantiationException {
```

Comment:
Can remove the `throws` clause and all exceptions

Reply:
Done